docs/faq.md

```diff
@@ -179,4 +180,22 @@ Also if you want to use the ts-node CLI you can execute TypeORM the following wa
 
 ```
 ts-node ./node_modules/bin/typeorm schema:sync
-```
\ No newline at end of file
+```
+
+## How to use Webpack for the backend?
+
+Webpack produces warnings due to what it views as missing require statements -- require statements for all drivers supported by TypeORM. To suppress these warnings for unused drivers, you will need to edit your webpack config file.
+
+```js
+const FilterWarningsPlugin = require('webpack-filter-warnings-plugin');
+
+module.exports = {
+    ...
+    plugins: [
+        //ignore the drivers you don't want. This is the complete list of all drivers -- remove the suppressions for drivers you want to use.
+        new FilterWarningsPlugin({
+            exclude: [/mongodb/, /mssql/, /mysql/, /mysql2/, /oracledb/, /pg/, /pg-native/, /pg-query-stream/, /redis/, /sqlite3/]
```

Comment:
what about mkdirp other non-driver modules? Don't we need them?

Reply:
Those loaded for me no problem, no errors. I don't know if they actually were needed for the switch statement, but it felt safer to put them in because PlatformTools.load() was getting called to load them. Loaded because they are standard nodejs modules rather than add ons like the drivers.

Reply:
Hmm strange you didnt had errors with `mkdirp`, its not a standard nodejs module....

Reply:
ah, okay so we have this package in dependencies, so you have it installed unlike driver packages